client/my-sites/plans/modernized-layout.tsx

```diff
@@ -134,10 +134,18 @@ const ModernizedLayout: React.FunctionComponent< { dropShadowOnHeader?: boolean
 			// this overrides the default .layout__content that adds unwanted padding
 			& .layout__content,
 			&.theme-default .focus-content .layout__content {
-				padding: ${ layoutContentPaddingTop } 0 0 0;
+				:not( .rtl ) & {
+					padding: ${ layoutContentPaddingTop } 0 0 0;
```

Comment:
I was wondering if we can use `padding-inline-` and `padding-block-` styles to simplify this? 🤔 

For example:
```css
padding: 0;
padding-block-start: ${ layoutContentPaddingTop };

@media ( min-width: ${ sidebarAppearanceBreakPoint } ) {
    padding-inline-start: calc( var( --sidebar-width-max ) + 1px );
}
```

Reply:
TIL! Thanks! that made the code way simpler :)